guides/migration/migrating-to-quarkus.adoc

```diff
@@ -68,3 +69,33 @@ While the WildFly distribution automatically discovered custom providers, even s
 
 Depending on what APIs your providers use you may also need to make some changes to the providers. If you only leveraged classes from Keycloak SPIs you shouldn't need to, but if you used other APIs from WildFly you may need to make some changes. In addition, JavaEE APIs like session/stateless beans are no longer supported.
 
+
+== Migrating using the Operator
+
+To use the Quarkus distribution on Kubernetes and OpenShift you need to use the new Operator, the https://github.com/keycloak/keycloak-operator[old Operator] does not support the new distribution.
+
+There is no "direct" migration path, to install Keycloak using the new Operator you need to create a new Custom Resource (CR) to end up with a new Keycloak Deployment based on the Quarkus distribution.
+
+The old and new operator can co-exists even in the same namespace since they are using a different API Group and Version in the CRDs.
```

Comment:
```suggestion
The old and new operator can co-exist even in the same namespace since they are using a different API Group and Version in the CRDs.
```